bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -3860,10 +3860,10 @@
         <sqlFile path="V1_98_WardsListSql.sql"/>
     </changeSet>
 
-    <changeSet id="bahmni-201703061748" author="Shruthi P, Pushpa">
+    <changeSet id="bahmni-202212131256" author="Shruthi P, Pushpa">
         <preConditions onFail="MARK_RAN">
-            <sqlCheck expectedResult="1">
-                SELECT count(property) FROM global_property WHERE property='emr.primaryIdentifierType';
+            <sqlCheck expectedResult="0">
```

Comment:
@rohit-yawalkar can we have a separate sql check for **bahmni.primaryIdentifierType**. this will be easy to understand.
<sqlCheck expectedResult="1">
                SELECT count(property) FROM global_property WHERE property='emr.primaryIdentifierType';
            </sqlCheck>

<sqlCheck expectedResult="0">
                SELECT count(property) FROM global_property WHERE property='bahmni.primaryIdentifierType';
            </sqlCheck>